### ISSUE TITLE
add on for Plotagon.

### ISSUE THREAD

**Issue by thunderbird5** [nvaccess/nvda#8001]
This is a fun and creative program that even the blind can use however, NVDA doesn't read the program at all. Is there anyone who would be able to create some sort of add on to get it working with NVDA? I can read the login page with the ocr but, that's about it.

I'm using the latest version of NVDA.

**Comment by Adriani90** [comment]
Please post request for addons on the addons list. If an addon is created, then the author could ofcourse raise a pull request against NVDA and cover specific issues. If you have specific issues with Plotagon (i.e. text selection etc.), please open individual issues for each problem. Thanks.
I am closing this for now and we can continue the discussion on the addons list.
